console/console-init/ui/src/AppRoutes.tsx

```diff
@@ -9,24 +9,26 @@ const getConnectionDetail = () => import("./Pages/ConnectionDetailPage");
 
 export const AppRoutes = () => (
   <SwitchWith404>
-    <Redirect path="/" to="/address-spaces" exact={true} />
+    <Redirect path="/" to="/address_spaces" exact={true} />
     <LazyRoute
-      path="/address-spaces"
+      path="/address_spaces"
       exact={true}
       getComponent={getAddressSpaceListPage}
     />
     <LazyRoute
-      path="/address-space/:id/:subList"
+      path="/address_space/name=:name&namespace=:namespace/:subList"
```

Comment:
I don't like this, `&` is used to separate params in the query string, I wonder if using it as part of the URL is even legal.

I'd rather use something like `/address-space/:namespace/:name`, that hints a clear hierarchy of the data 

I'd also use `-` instead of `_` but that's almost a personal preference. There are SEO concerns to be aware of when using underscores](https://www.woorank.com/en/blog/underscores-in-urls-why-are-they-not-recommended) that don't really apply to this app, but I think it's better to follow best practices nonetheless.